src/content/guides/farmlands-portal.mdx

```diff
@@ -190,6 +190,58 @@ On the Members page you can manage the members connected to the Centrapay Busine
 
 ![Centrapay business portal members page](/centrapay-portal-members.png)
 
+**Adding members**
+
+1. An Account owner can **select the Members page** in the left sidebar menu.
+![Select the Members page](/members/members-page.png)
+
+2. The Account owner can click **Add people**.
+![Click Add People](/members/add-people.png)
```

Comment:
I think we should be more descriptive in the image alt text.
Maybe Centrapay business portal add members screen.

Up to you :)